config/connector_options.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+type ConnectorOptions func(*options)
+
+type options struct {
```

Comment:
would it make sense to reuse those from `sqlcon`?

Reply:
That makes sense to me

Reply:
Re: `OmitSQLArgsFromTracingSpans` I was following some of the concerns that Google identified in [Dapper](https://ai.google/research/pubs/pub36356) for not releasing potentially sensitive details in traces. Perhaps in some organizations, the engineers that have access to look at traces should not be able to view such data for compliance/privacy reasons.

I can remove this feature until we get a feature request, maybe?

Reply:
My apologies for not being clear on that - I meant that traces should always omit the data. For general hydra use-cases, I don't think there's a time this is needed outside of debugging/testing.

Reply:
Sweet, that sounds good to me. I will leave this option on by default and remove the config and if people ask for it be included, they can create an issue so we can re-introduce it.